connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -61,7 +61,7 @@ message Command {
 // almost oblivious to the server-side behavior.
 message SqlCommand {
   // (Required) SQL Query.
-  string sql = 1;
+  string sql = 1 [deprecated=true];
```

Comment:
@zhengruifeng where am I supposed to set the sql text now?

Reply:
```
  // (Optional) The relation that this SQL command will be built on.
  Relation input = 6;
```

this Relation can be a `message SQL` or `message WithRelations`